.lintstagedrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  "*.{js,jsx,ts,tsx}": ["eslint --fix", "prettier --write"],
```

Comment:
What's the reasoning behind removing eslint fix from the root package lint-staged config?

Reply:
We neither have js/ts files in the root nor do we have an eslint-config, why should we want lint-staged to execute eslint there? In my understanding, with the package-level lintstaged-files we have now, the eslint is executed in separate processes where needed.

Reply:
Not today, but is there any harm in keeping the line there? I.e. if no files of that type are found, it has not effect, and if we one day end up adding one of those files, then we'd want to lintstaged action to run anyway, no?